pkg/topology/test/helper.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package test
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+// RandomAddressAt generates a random address
+// at proximity order prox relative to address
+// if prox is negative a random address is generated.
+func RandomAddressAt(self swarm.Address, prox int) swarm.Address {
```

Comment:
It would be nice to have a test for this function. Just to check if expected proximity is correct.

Reply:
it was a fun one :+1: please double check